scene/gui/tabs.cpp

```diff
@@ -184,10 +184,7 @@ void Tabs::_gui_input(const Ref<InputEvent> &p_event) {
 			}
 
 			int found = -1;
-			for (int i = 0; i < tabs.size(); i++) {
```

Comment:
Could you check if offset can be negative ? If so, it's possible that this creates a different behavior.
In such case, the condition would be replaced by:
```
for (int i = max(0, offset); i < tabs.size(); i++) {
```
to keep the same behavior.

Reply:
https://github.com/godotengine/godot/blob/master/scene/gui/tabs.cpp#L122

https://github.com/godotengine/godot/blob/master/scene/gui/tabs.cpp#L179

https://github.com/godotengine/godot/blob/master/scene/gui/tabs.cpp#L862